.markdownlint.yaml

```diff
@@ -50,4 +50,8 @@ MD040: false
 MD041: true
 
 # MD046/code-block-style
-MD046: false
\ No newline at end of file
+MD046: false
+
+# MD050/strong-style 
+MD050:
+  style: "asterisk"
```

Comment:
ah, true, let me remove it

Reply:
ah, it is already failed. So, it does not matter for what reason it would fail..

Reply:
yea, but even then it does not change anything, as this file is generated after main pipeline runs, so it will fail for both cases.

Maybe just worth ading it for other files? but for that one the pipeline will fail in PR, so also not a big benefit.

Reply:
yea, that is the [default](https://github.com/DavidAnson/markdownlint/blob/v0.36.1/doc/md050.md) option, 

```
MD050:
  style: "consistent"
```

so there are 2 possibilities: 
 1. we add this rule and it will fail when underscores( _ ) appears due to incorrect sign being used as bolder sign, 
 2. we do not add it and it will fail when underscores( _ ) due to inconsistent bolder signs